packages/reference/src/resources/useResourceLinkActions.ts

```diff
@@ -26,20 +25,17 @@ export function useResourceLinkActions({
   const multiple = field.type === 'Array';
 
   const toLinkItem = useMemo(() => {
-    function toUrn(entry: EntryProps<KeyValueMap>) {
-      return `crn:${apiUrl}:::content:spaces/${entry.sys.space.sys.id}/entries/${entry.sys.id}`;
-    }
-
     return (entry: EntryProps<KeyValueMap>) => {
```

Comment:
This can be moved outside the hook now. Let's also bump `contentful-management` and type the `entry` param properly.

Reply:
Alright I was waiting for the type to be ready. With this `This can be moved outside the hook now` do you mean that I should extract it from the hook and put it on top the file as a function?